tests/pytests/unit/test_payload.py

```diff
@@ -210,3 +213,93 @@ def test_constants():
     sdata = salt.payload.dumps(constant)
     odata = salt.payload.loads(sdata)
     assert odata == constant
+
+
+def test_package():
+    value = salt.utils.msgpack.dumps("test")
+    assert value == salt.payload.package("test")
```

Comment:
```suggestion
    assert salt.payload.package("test") == VALUE
```
Assert what we get against what we expect(makes more sense when it fails and pytest shows the diff)

Same for the remaining tests.